From ede0fa7d410be407164b68570a1540378bf0bd09 Mon Sep 17 00:00:00 2001
From: Thomas Petazzoni <thomas.petazzoni@free-electrons.com>
Date: Sat, 19 Jul 2014 10:40:59 +0200
Subject: [PATCH] Remove installation of docs/examples

They get installed in the wrong directory, i.e if DESTDIR is
/path/to/output/target, they will be installed in
/path/to/output/target./examples and /path/to/output/target./docs.

Since we don't need docs and examples in Buildroot anyway, simply get
rid of them.

Signed-off-by: Thomas Petazzoni <thomas.petazzoni@free-electrons.com>
---
 Makefile.am | 11 -----------
 1 file changed, 11 deletions(-)

diff --git a/Makefile.am b/Makefile.am
index c56860a..6b1267b 100644
--- a/Makefile.am
+++ b/Makefile.am
@@ -69,14 +69,3 @@ liblcdapi_la_CXXFLAGS = \
 	-g0 -DNODEBUG -Os
 
 ACLOCAL_AMFLAGS = -I m4
-
-docsdir = $(top_srcdir)/docs
-exampledir = $(top_srcdir)/example
-dist_docs_DATA = \
-	docs/Makefile.am \
-	docs/Makefile.in \
-	docs/Doxyfile.in
-dist_example_DATA = \
-	example/client.cpp \
-	example/Makefile.am \
-	example/Makefile.in
-- 
2.0.0

